app/src/main/java/com/example/keybindhelper/Adapters/KeybindAdapter.kt

```diff
@@ -79,7 +79,7 @@ class KeybindAdapter(private val keybindList: MutableList<Keybind>) :
         val d = Dialog(c)
         d.requestWindowFeature(Window.FEATURE_NO_TITLE)
         d.setCancelable(true)
-        d.setContentView(R.layout.edut_keybind_dialog_2)
+        d.setContentView(R.layout.edit_keybind_dialog_2)
         val nameEt = d.findViewById<TextInputEditText>(R.id.edit_keybind_name)
```

Comment:
Changed the typo, I hope that was not a planned name?

Reply:
This was suggested in a previously-submitted code review.  When that occurs, only the first in wins.